README.md

```diff
@@ -11,93 +11,124 @@ Master (Release): [![Build Status](https://jenkins.qa.cancercollaboratory.org/bu
 
 ## Documentation
 
-Explore documentation with the Arranger [Read the Docs](https://arranger.readthedocs.io/en/latest/index.html).
+This file is meant as a quick introduction, but for more in-detail documentation, you should explore Arranger's "[Read the Docs](https://arranger.readthedocs.io/en/latest)". If interested, see our Open Source [License](https://github.com/overture-stack/arranger/blob/master/LICENSE)
 
-### Quickstart
+- [Getting Started](#getting-started)
+  - [Development Setup](#--development-setup)
+  - [Dockerized Setup (a.k.a. Quickstart)](#--dockerized-setup)
+- [Motivation](#motivation)
+- [What is a "Data Portal"?](#data-portal)
+- [Roadmap](#roadmap)
+- [Development Details](#development-details)
 
-#### Clone, Install, Bootstrap
+---
 
-```
-git clone git@github.com:overture-stack/arranger.git
-cd arranger
-npm i
-npm run bootstrap
+### Getting Started
 
-# watch modules and rebuild on change
-npm run watch
+#### - Development Setup
 
-# run tests for all modules
-npm test
+Setting up the project, and ready things to make changes
 
-# run server on :5050
-npm run server
+```bash
+# 1. clone the repository
+  git clone git@github.com:overture-stack/arranger.git
+
+# 2. enter the project's folder
+  cd arranger
 
-# run dashboard on :6060
-npm run dashboard
+# 3. install the dependencies
+  npm i
 
-# run portal on :7070
-npm run portal
+# 4. install the module's own dependencies
+  npm run bootstrap
 
-# run storybook on :8080
-npm run storybook
 ```
 
-- [Documentation](#Documentation)
-- [Motivation](#motivation)
-- [What is a "Data Portal"?](#data-portal)
-  - [Topology](#topology)
-- [Roadmap](#roadmap)
-- [Development](#development)
-- [License](https://github.com/overture-stack/arranger/blob/master/LICENSE)
+Now you should be able to start the following processes from the project's root:
```

Comment:
Suggest adding "...from the project's root **folder**:" for extra clarity

Reply:
```suggestion
Now you should be able to start the following processes from the project's root folder:
```